jest.config.ts

```diff
@@ -4,10 +4,15 @@ const config: Config.InitialOptions = {
   preset: 'ts-jest',
   testEnvironment: 'node',
   roots: ['src'],
-  globals: {
-    'ts-jest': {
-      tsconfig: 'tsconfig.test.json',
-    },
+  transform: {
+    '^.+\\.test.ts$': [
+      'ts-jest',
+      {
+        useESM: true,
```

Comment:
```suggestion
```
I think we only want to use this if this package starts supporting ESM, can probably just remove for now